extensions/wikia/ExactTargetUpdates/client/builders/UpdateRequestBuilder.class.php

```diff
@@ -9,8 +9,21 @@ class UpdateRequestBuilder extends BaseRequestBuilder {
 
 	private $userData;
 	private $edits;
-
-	private static $supportedTypes = [ self::PROPERTIES_TYPE, self::EDITS_TYPE, self::USER_TYPE ];
+	private $wikiData;
+
+	private static $supportedTypes = [
+		self::PROPERTIES_TYPE,
+		self::EDITS_TYPE,
+		self::USER_TYPE,
+		self::WIKI_TYPE,
+		self::WIKI_CAT_TYPE
+	];
+
+	public function withWikiData( $wikiId, array $wikiData ) {
+		$this->wikiData = $wikiData;
+		$this->wikiId = $wikiId;
+		return $this;
+	}
 
 	public function withUserData( array $userData ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "$userData" which has the same name as the field declared at line 10. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1117)